src/main/java/no/digipost/api/client/DigipostClientMock.java

```diff
@@ -58,34 +78,59 @@ public DigipostClientMock(ApiFlavor apiFlavor) {
 		}, apiService);
 	}
 
+	public void start(){
+		KeyPair keyPair = getKeyPair(KEY_STORE_ALIAS, KEY_STORE_PASSWORD);
+		digipostApiMock.start(PORT, requestsAndResponsesMap, keyPair);
+	}
+
+	public static KeyPair getKeyPair(final String alias, final String password) {
+		try {
+			KeyStore keystore = KeyStore.getInstance(KeyStore.getDefaultType());
+			keystore.load(DigipostClientMock.class.getClass().getResourceAsStream("/mockKeystore.jks"), KEY_STORE_PASSWORD.toCharArray());
+
+			final Key key = (PrivateKey) keystore.getKey(alias, password.toCharArray());
```

Comment:
Hva med å gjøre `key` til en `PrivateKey` her, så slipper vi å caste også på linje 95?
Og kanskje fjerne final på denne variabelen og de to under. Gir ikke all verdens verdi